packages/thirdweb/src/utils/any-evm/zksync/isZkSyncChain.ts

```diff
@@ -1,12 +1,24 @@
+import { withCache } from "../../promise/withCache.js";
 import type { Chain } from "../../../chains/types.js";
 
-export function isZkSyncChain(chain: Chain) {
-  return (
-    chain.id === 324 ||
-    chain.id === 300 ||
-    chain.id === 302 ||
-    chain.id === 11124 ||
-    chain.id === 282 || // cronos zkevm testnet
-    chain.id === 388 // cronos zkevm mainnet
+export async function isZkSyncChain(chain: Chain) {
+  const stack = await getChainStack(chain.id);
+
+  return stack === "zksync-stack";
+}
+
+export async function getChainStack(chainId: number): Promise<string> {
```

Comment:
This probably doesn't need to be exported, don't think you're using it anywhere

Reply:
I can do u one better, lint is not allowing you to export it SINCE you are not using it anywhere

Reply:
fixed